pkg/apis/streaming/v1alpha1/pulsarprovider_defaults.go

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2019 the original author or authors.
```

Comment:
Please preserve the [existing copyright header format](https://github.com/projectriff/system/blob/master/hack/boilerplate.go.txt) x(all new go files)